src/config.rs

```diff
@@ -31,6 +31,7 @@ pub struct Config {
     flush_every_ms: Option<u64>,
     snapshot_after_ops: usize,
     snapshot_path: Option<String>,
+    cache_fixup_threshold: usize,
```

Comment:
this is the number of unmerged partial pages in cache that can exist before an attempt to squish them together into a cached `MergedResident`